subcommand_plugin_examples/c_subcommand_plugin_tutorial/README.md

```diff
@@ -0,0 +1,243 @@
+
+The `temp_conv_c.py` module can then call the C function using the format `CDLL(so_file)` from inside of the `conda_temp_converter()` function. Once that code is in place, then the `conda_temp_converter()` function can be registered via the plugin manager hook called `conda_subcommands` using the `@conda.plugins.register` decorator:
+
+```python
+# temp_conv_c.py
+
+from ctypes import CDLL
+import conda.plugins
+
+
+def conda_temp_converter(*args, **kwargs):
+    so_file = "/Users/biancahenderson/Documents/GitHub/conda/temp_converter/c_to_f.so"
+    my_functions = CDLL(so_file)
+    return my_functions.main()
+
+
+@conda.plugins.register
+def conda_subcommands():
+    yield conda.plugins.CondaSubcommand(
+        name="temp-converter",
+        summary="A custom subcommand written in C that converts Celsius to Fahrenheit",
+        action=conda_temp_converter,
+    )
+```
+
+## Packaging the custom subcommand
+
+In order to install the `conda temp-converter` custom subcommand we will need to configure a Python build system. You can either use the [PEP 621][pep 621] compliant `pyproject.toml` or alternatively `setup.py` can be used (not shown in this tutorial):
+
+```toml
+[build-system]
+requires = ["setuptools>=61.0", "setuptools-scm"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "temp-converter"
+version = "1.0"
+description = "A custom subcommand written in C that converts Celsius to Fahrenheit"
+requires-python = ">=3.7"
+dependencies = ["conda"]
+
+[tools.setuptools]
+py_modules=["temp-converter"]
+
+[project.entry-points.conda]
+temp-converter = "temp_conv_c"
+```
+
+<details>
+<summary><code>pyproject.toml details</code></summary>
+
+> #### `[build-system]`
+> - `requires` This is a list of requirement specifiers for build-time dependencies of a package.
+> - `build-backend` Build backends have the ability to accept configuration settings, which can change the way that the package building is handled.
+> 
+> #### `[project]`
+> * `name` (required) This is the name of the package that contains your subcommand. This is also how others will find your subcommand package if you choose to upload it to PyPI.
+> * `version` (required) The version of the project; can be specified *either* statically or listed as dynamic.
+> `description` A brief description of the project.
+> * `requires-python` The version(s) of Python required by your project.
+> * `dependencies` These are all of the dependencies for your project. This specific subcommand example requires `conda`, which is why it is listed here.
+>
+> For more information on `pyproject.toml` see the [PyPA packaging documentation][pyproject.toml docs].
+
+</details>
+
+
+> **Note:**
+> For more information about entry points specification in general, please read [PyPA's entrypoints documentation][entrypoints docs].
+
+### Development/Editable Install
```

Comment:
```suggestion
### Development/editable install
```

Let's keep the capitalizing the first letter here. That's typically how article sub-sections are handled.